_config.yml

```diff
@@ -44,3 +44,51 @@ officers:
     twitter: "awflwafl"
     linkedin: "artemiy-kondratiev"
     github: "awflwafl"
+membership_tiers:
```

Comment:
A future enhancement here could be to pull these from Nexudus's [public API for price plans](https://developers.nexudus.com/reference/get-public-plans) on page load, which would mean we wouldn't have to keep this in sync with Nexudus.